01-Login/README.md

```diff
@@ -1,84 +1,705 @@
+
+::: note
+If you download the sample from the top of this page these details are filled out for you.
+
+If you have more than one application in your account, the sample comes with the values for your **Default App**.
+:::
+
+### Configure Callback URLs
+
+A callback URL is a URL in your application where Auth0 redirects the user after they have authenticated. 
+
+The callback URL for your app must be whitelisted in the **Allowed Callback URLs** field in your [Application Settings](${manage_url}/#/applications/${account.clientId}/settings). If this field is not set, users will be unable to log in to the application and will get an error.
+
+::: note
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Callback URL** to `http://localhost:3000/callback`.
+:::
+
+### Configure Logout URLs
+
+A logout URL is a URL in your application that Auth0 can return to after the user has been logged out of the authorization server. This is specified in the `returnTo` query parameter.
+
+The logout URL for your app must be whitelisted in the **Allowed Logout URLs** field in your [Application Settings](${manage_url}/#/applications/${account.clientId}/settings). If this field is not set, users will be unable to log out from the application and will get an error.
+
+::: note
+If you are following along with the sample project you downloaded from the top of this page, the logout URL you need to whitelist in the **Allowed Logout URLs** field is `http://localhost:3000`.
+:::
+
+### Configure Allowed Web Origins
+
+You need to whitelist the URL for your app in the **Allowed Web Origins** field in your [Application Settings](${manage_url}/#/applications/${account.clientId}/settings). If you don't whitelist your application URL, the application will be unable to automatically refresh the authentication tokens and your users will be logged out the next time they visit the application, or refresh the page.
+
+::: note
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Web Origins** to `http://localhost:3000`.
+:::
+
+## Integrate Auth0 in your Application
+
+### Loading auth0.js
+
+You need the auth0.js library to integrate Auth0 into your application.
+You can either install the library locally in your application or load it from CDN.
+
+### Loading via dependencies
+
+Install auth0.js using npm or yarn.
 
 ```bash
-npm install
+# installation with npm
+npm install --save auth0-js
+
+# installation with yarn
+yarn add auth0-js
+```
+
+Once you install auth0.js, add it to your build system or bring it in to your project with a script tag.
+
+```html
+<script type="text/javascript" src="node_modules/auth0-js/build/auth0.js"></script>
 ```
 
-### Configuration
+### Loading it from CDN
+ 
+If you do not want to use a package manager, you can retrieve auth0.js from Auth0's CDN.
+
+```html
+<script src="${auth0js_url}"></script>
```

Comment:
This will need to be replaced with the actual CDN url, as the `auth0js_url` variable doesn't mean anything in GitHub.